### PR TITLE
Bump fluentd version to 1.11.2

### PR DESCRIPTION
**Pull request by elsesiy**
Signed-off-by: Jonas-Taha El Sesiy <github@elsesiy.com>

### CODE REVIEW

**Comment by elsesiy** [comment]
cc @repeatedly @cosmo0920

File: Makefile [hunk 1]
```diff
@@ -12,22 +12,22 @@
...
-	v1.11/debian-syslog:v1.11.1-debian-syslog-1.0,v1.11-debian-syslog-1 \
-	v1.11/debian-forward:v1.11.1-debian-forward-1.0,v1.11-debian-forward-1 \
-	v1.11/debian-gcs:v1.11.1-debian-gcs-1.0,v1.11-debian-gcs-1 \
-	v1.11/debian-graylog:v1.11.1-debian-graylog-1.0,v1.11-debian-graylog-1 \
-	v1.11/debian-papertrail:v1.11.1-debian-papertrail-1.0,v1.11-debian-papertrail-1 \
-	v1.11/debian-logzio:v1.11.1-debian-logzio-1.0,v1.11-debian-logzio-1 \
-	v1.11/debian-kafka:v1.11.1-debian-kafka-1.0,v1.11-debian-kafka-2 \
-	v1.11/debian-kinesis:v1.11.1-debian-kinesis-1.0,v1.11-debian-kinesis-1
+	v1.11/debian-azureblob:v1.11.2-debian-azureblob-1.0,v1.11-debian-azureblob-1 \
+	v1.11/debian-elasticsearch7:v1.11.2-debian-elasticsearch7-1.3,v1.11-debian-elasticsearch7-1,v1-debian-elasticsearch \
```
Comment (cosmo0920) [inline Makefile:16]: When we upgrade Fluentd version, we should use `1.0` for patch Docker image version.

```suggestion
	v1.11/debian-elasticsearch7:v1.11.2-debian-elasticsearch7-1.0,v1.11-debian-elasticsearch7-1,v1-debian-elasticsearch \
```

File: Makefile [hunk 2]
```diff
@@ -12,22 +12,22 @@
...
+	v1.11/debian-elasticsearch6:v1.11.2-debian-elasticsearch6-1.0,v1.11-debian-elasticsearch6-1 \
+	v1.11/debian-loggly:v1.11.2-debian-loggly-1.0,v1.11-debian-loggly-1 \
+	v1.11/debian-logentries:v1.11.2-debian-logentries-1.0,v1.11-debian-logentries-1 \
+	v1.11/debian-cloudwatch:v1.11.2-debian-cloudwatch-1.1,v1.11-debian-cloudwatch-1 \
```
Comment (cosmo0920) [inline Makefile:20]: Ditto.

```suggestion
	v1.11/debian-cloudwatch:v1.11.2-debian-cloudwatch-1.0,v1.11-debian-cloudwatch-1 \
```

File: Makefile [hunk 3]
```diff
@@ -12,22 +12,22 @@
...
+	v1.11/debian-stackdriver:v1.11.2-debian-stackdriver-1.0,v1.11-debian-stackdriver-1 \
+	v1.11/debian-s3:v1.11.2-debian-s3-1.1,v1.11-debian-s3-1 \
```
Comment (cosmo0920) [inline Makefile:22]: Ditto.

```suggestion
	v1.11/debian-s3:v1.11.2-debian-s3-1.0,v1.11-debian-s3-1 \
```

**Comment by elsesiy** [comment]
@cosmo0920 I updated the PR

**Pull request merged**